main/src/routes/indexRouter.js

```diff
@@ -1,5 +1,4 @@
 var express = require('express');
-const {Main} = require("../Main");
```

Comment:
Ah yes good catch, I only caught this when I added eslint https://github.com/ericrkuo/Flow/pull/29. Maybe we can merge in the eslint PR first so that you can check any linting errors?

Reply:
Hmm, I can do it in a separate PR in case there's a lot of changes?

Reply:
Ideally, we want the code we merge to be linted already, not in a separate PR. There shouldn't be a lot of changes, essentially all you need to do is merge in the code, run the lint fix command, and check if there are any lint errors u needa resolve urself!